### DISCUSSION TITLE
2倍速播放时较大概率声音卡顿

### DISCUSSION THREAD

**Discussion by 21floatdream** [ywmoyue/biliuwp-lite#344] [bug]
### Bug 描述

2倍速播放视频，画面无问题，但是声音会出现“跳跃”，声音过一秒卡一下就像剪辑了一样

### 操作步骤

```plain text
1.打开一个长点的视频并开启2倍速
2.正常播放几分钟
3.几分钟后出现“跳读”，且画面正常没有“错误已记录”的提示
4.bug出现后改变倍速如1.25；1.5后，再切回到2倍速回复正常
5.几分钟后又出现。
```

### 期望结果

声音正常播放

### 日志

[可能没用，bug出现时没有“错误已记录”]
[Uploading 20231011.log…]()

### 应用来源

Github/其他Github镜像站

### 安装方式

appx直接安装

### 系统版本

Windows 10 21H1 (Build 19043)

### 设备类型

其他

### 设备架构

x64

### 应用架构

x64

### 备注

我的设备比较特殊，是小米平板2，装的win，Windows 10 专业版
处理器Intel(R) Atom(TM) x5-Z8500  CPU @ 1.44GHz   1.44 GHz。
可能是我设备原因，后续会在笔记本测试

### 代码贡献

我更擅长发现问题

**Comment by GD-Slime** [comment]
win10 最好按照安装包中的说明用脚本来装，不然可能会有bug
不过也不排除是特殊的设备原因
